src/app/tracing/util/map-constants.ts

```diff
@@ -0,0 +1,26 @@
+import { MapType, TileServer } from "../data.model";
+import { COLORS } from "./colors";
+
+export const MAP_CONSTANTS = {
+    types: [
+        MapType.MAP_ONLY,
+        MapType.SHAPE_ONLY,
+        MapType.TILES_AND_SHAPE,
+    ] as Array<MapType>, // please note: array - so order is relevant!
+    tiles: [
+        TileServer.MAPNIK /*TileServer.BLACK_AND_WHITE*/,
+    ] as Array<TileServer>, // please note: array - so order is relevant!
+    labels: {
+        [TileServer.MAPNIK]: "Mapnik",
+        [MapType.SHAPE_ONLY]: "Shape File",
+        [MapType.TILES_AND_SHAPE]: "Map & Shape File",
+        //temporarily deactivated, therefore just a comment and not removed
+        //[TileServer.BLACK_AND_WHITE]: "Black & White",
+    } as Record<TileServer | MapType, string>,
+    defaults: {
+        mapType: MapType.MAP_ONLY,
+        tileServer: TileServer.MAPNIK,
+        geojsonBorderWidth: 0.5,
+        geojsonBorderColor: COLORS.primary_dark,
```

Comment:
I would rather use `COLORS.black` instead of `COLORS.primary_dark` because this color should be independent of the color scheme.

Reply:
@ruegenbfr I actually disagree slightly here. 
`primary_dark` should always be a black or off-black and should match the color of our text-labels on the graph. 
If we decide in the future to use a slightly softer tone of black, it would be preferable if the lines change alongside the text.

On that note, though, I am uncertain whether our graph lines are drawn with `COLORS.black` or `COLORS.primary_dark`. It should probably be the latter.

It's not worth complicating this ticket over, but this is my general opinion. What do you think?

Reply:
Actually, I might be thinking too far ahead here, especially since we are not really planning much major CSS work anymore in the coming months. Though, I am hoping we get the time for a big CSS polish after the import features and before the project is completed.